hedyorg/hedy
# Avoid double/triple/n print on aborted program that contains both ask & print

- Closes #820 
- Implements a count of how many programs are currently executed. If the count is more than 1, the `outf` is disabled.
- This prevents the printing of the output of a program that, containing `ask` and `print`, is edited while awaiting input, and thus discarded.
- I explored the alternative of killing/disabling the interruption of a certain program in Skulpt, to no avail.

**How to test**:
- Open http://localhost:5000/hedy?lang=en
- Enter the following program
```
ask foo
print a
```
- Execute the program
- When prompted for input, change the program to:
```
ask foo
print b
```
- Note that the input box should be gone. Now run the program. You should only see `b` in the output, rather than `a`, followed by `b`.
- To test the case of a triple print, simply edit the program to `print c` before running it.
